### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#823]
# Reference
- [ ] [BatchNormalization、LayerNormalization、InstanceNorm、GroupNorm、SwitchableNorm总结](https://blog.csdn.net/liuxiao214/article/details/81037416)

# Brief

Normalization | Paper & Code
-- | --
Batch Normalization | -  [paper - 2015 -Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/pdf/1502.03167.pdf)
Layer Normalizaiton | -  [paper - 2016 - Layer Normalization](https://arxiv.org/pdf/1607.06450.pdf)
Instance Normalization | -  [paper - 2016 - Instance Normalization: The Missing Ingredient for Fast Stylization](https://arxiv.org/pdf/1607.08022.pdf) <br>-  [Code](https://github.com/DmitryUlyanov/texture_nets)
Group Normalization | -  [paper - 2018 - Group Normalization](https://arxiv.org/pdf/1803.08494.pdf)
Switchable Normalization | -  [paper - 2018 - Differentiable Learning-to-Normalize via Switchable Normalization](https://arxiv.org/pdf/1806.10779.pdf) <br>-  [Code](https://github.com/switchablenorms/Switchable-Normalization)
